config/config/sc-config.yaml

```diff
@@ -10,6 +10,35 @@ objectStorage:
     influxDB: ${CK8S_ENVIRONMENT_NAME}-influxdb
     scFluentd: ${CK8S_ENVIRONMENT_NAME}-sc-logs
 
+  ## Off-site backup replication between two providers or regions using rclone sync
+  sync:
+    enabled: false
+    # dryrun: false
+
+    ## Options are 's3'
+    type: none
+    # s3:
+    #   region: set-me
+    #   regionEndpoint: set-me
+    #   # Generally false when using AWS and Exoscale and true for other providers.
+    #   forcePathStyle: set-me
+    #   # v2Auth: false
+
+    ## Buckets to sync.
+    buckets: []
```

Comment:
Would it make sense to have a `syncDefaultBuckets` option that would add all the buckets from the `objectStorage.buckets` list? If it's too cumbersome, then we can have that as a separate issue.

Reply:
Yes, I can add that.